provider/oci/instance.go

```diff
@@ -224,11 +226,10 @@ func (o *ociInstance) deleteInstance() error {
 		if o.isTerminating() {
 			break
 		}
-		if iteration >= 30 && o.raw.LifecycleState == ociCore.InstanceLifecycleStateRunning {
-			logger.Warningf("Instance still in running state after %v checks. breaking loop", iteration)
-			break
+		if iteration >= maxPollIterations && o.raw.LifecycleState == ociCore.InstanceLifecycleStateRunning {
+			return errors.Errorf("Instance still in running state after %v checks", iteration)
 		}
-		<-o.env.clock.After(1 * time.Second)
+		<-o.env.clock.After(2 * time.Second)
```

Comment:
Why is this poll time 2s, while the other one is 1s, but we use the same pollIterations in both places?